templates/admin/dashboard.html.twig

```diff
@@ -0,0 +1,94 @@
+{% extends 'base.html.twig' %}
+
+{% block main_content %}
+	{% include 'layout/header.html.twig' with ({
+		'title': "Profil de #{user.displayName}",
+		'subtitle': ''
+	}) %}
+
+
+	<div class="dashboard-container">
+		<div class="member-avatar" style="background-image: url({{ user.avatar ?? '/media/layout/icons/member_default.png' }})"></div>
+			<h3>Informations</h3>
+			<ul>
+				<li><span class="bolder">Pseudo : </span>{{ user.displayName }}</li>
+				<li><span class="bolder">Nom complet : </span>{{ user.name }}</li>
+				<li><span class="bolder">Email : </span>{{ user.email }}</li>
+				<li><span class="bolder">Commune : </span>{{ user.locality }}</li>
+				<li><span class="bolder">Code postal : </span>{{ user.postCode }}</li>
+				<li><span class="bolder">Pays : </span>{{ user.country }}</li>
+				<li><span class="bolder">Type de profil : </span>{{ user.profileType }}</li>
+				<li><span class="bolder">{{ user.isNewsletterSubscriber ? 'Reçoit' : 'Ne reçoit pas' }} notre lettre d'actualités</span></li>
+				<li><span class="bolder">{{ user.isMailsSubscriber ? 'Accepte' : 'N’accepte pas'}} d'être contacté(e) par mail</span></li>
+			</ul>
+
+		<a href="" class="button bt-add edit open" data-open="admin-user" data-user="{{ user|getJsonSerializedEditUserProfile }}">Modifier ses paramètres</a>
+        <a href="{{ path('admin_user_password_edit', {userId:user.id}) }}" class="button bt-add">Modifier son mot de passe</a>
+
+		<h3>ses dernières contributions aux stations</h3>
+		<div class="dashboard-text">Les stations auxquelles {{user.displayName}} a contribué ou qu’il a créées :</div>
+		<div class="cards-grid-container">
+			{% set cards = setStationCards(stations) %}
+			{%  for card in cards %}
+				{% include 'components/cards/station-cards.html.twig' with {
+					station: card.station,
+					observations: card.observations,
+					canAdminStation: true
+				} %}
+			{% endfor %}
+		</div>
+		<h3>ses dernières observations</h3>
+		{% set periodsNames = constant('App\\Entity\\Event::DISPLAY_LABELS') %}
+		{% set pictureUrl = '' %}
+		<div class="dashboard-obs-info-container">
+			{% for observation in observations %}
```

Comment:
this for logic is the same in dashboard.html.twig and in stations-cards.html.twig, maybe a twig filter could be useful here